cluster/store/store.go

```diff
@@ -534,7 +534,21 @@ func (st *Store) Stats() map[string]any {
 	// If the raft stats exist, add them as a nested map
 	if st.raft != nil {
 		stats["raft"] = st.raft.Stats()
+		// add the servers information
+		var servers []map[string]any
+		if cf := st.raft.GetConfiguration(); cf.Error() == nil {
+			servers = make([]map[string]any, len(cf.Configuration().Servers))
+			for i, server := range cf.Configuration().Servers {
+				servers[i] = map[string]any{
+					"id":       server.ID,
+					"address":  server.Address,
+					"suffrage": server.Suffrage,
+				}
+			}
+			stats["raft_latest_configuration_servers"] = servers
```

Comment:
**suggestion:** do we need it ? , i think it's provided in `raft.Stats()` ?
w.r.t https://github.com/hashicorp/raft/blob/341ea38693b41933add6f24a9f544702ffe13391/api.go#L1171

Reply:
Yes it's there, but it looks like this:
```
"latest_configuration": "[{Suffrage:Voter ID:weaviate-1 Address:10.244.0.10:8300} {Suffrage:Voter ID:weaviate-0 Address:10.244.0.13:8300} {Suffrage:Nonvoter ID:weaviate-3 Address:10.244.0.12:8300} {Suffrage:Nonvoter ID:weaviate-2 Address:10.244.0.11:8300}]",
```
I have changed this so we would get a proper json array with objects in response like so:
```
"latestConfiguration": [
    {
        "address": "10.244.0.22:8300",
        "id": "weaviate-1",
        "suffrage": 0
    },
    {
        "address": "10.244.0.19:8300",
        "id": "weaviate-2",
        "suffrage": 0
    },
    {
        "address": "10.244.0.23:8300",
        "id": "weaviate-0",
        "suffrage": 0
    }
],
```